parsers/typespec.c

```diff
@@ -388,8 +390,12 @@ static void parseNamespace(tokenInfo *const token)
 	vStringDelete(name);
 }
 
-static void parseEnum(tokenInfo *const token)
+static void parseEnum(tokenInfo *const parentToken)
 {
+	tokenInfo *token = newToken ();
```

Comment:
I cannot find a line to delete this new token.
I expect Valgrind on the CI may say something.